package.json

```diff
@@ -15,15 +15,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@bigcommerce/big-design": "^0.27.0",
+    "@bigcommerce/big-design": "^0.30.2",
```

Comment:
We should be including the other packages too instead of relying on hoisting here.

Reply:
the only other package I see is icons - I'm happy to include, though that's not necessarily within the purview of this PR (whose intent was to update to npm 7+). If we want to update/ modify / expand `package.json`, including a newer version of Next, that should be a separate PR.

Reply:
Yeah let's do it in another PR, should've added a 🍹 here.

I added this comment because when you install `@bigcommerce/big-design` it'll install both `@bigcommerce/big-design-icons` and `@bigcommerce/big-design-theme` as hoisted dependencies, since the core package depends on the two. I believe #70 works because it uses the hoisted dependency, but we should be explicitly defining `@bigcommerce/big-design-theme` in `dependencies` (I'll wait for this PR to add it).